Cargo.toml

```diff
@@ -49,7 +49,7 @@ members = [
 ]
 
 [workspace.package]
-version = "0.2.0-alpha"
+version = "1.0.0-alpha"
```

Comment:
BTW, do we remember to update the version to `0.18.0` when releasing? 🥵

Reply:
> BTW, do we remember to update the version to `0.18.0` when releasing? 🥵

I will update the version in release script before next release.

Reply:
```
toml set --toml-path Cargo.toml workspace.package.version ${BUILDKITE_TAG#*v}
```
Already update in release script.